examples/src/main/python/ml/decision_tree_classification_example.py

```diff
@@ -65,8 +67,9 @@
     predictions.select("prediction", "indexedLabel", "features").show(5)
 
     # Select (prediction, true label) and compute test error
-    evaluator = MulticlassClassificationEvaluator(
-        labelCol="indexedLabel", predictionCol="prediction", metricName="accuracy")
```

Comment:
`pep8` tool can automatically do this return if line is > 100 char. There is indeed no preference between this format and:

    evaluator = MulticlassClassificationEvaluator(labelCol="indexedLabel",
                                                  predictionCol="prediction",
                                                  metricName="accuracy")